### PR TITLE
Make sure test reporter prints out stderr, and disables Bigquery for non-contributors

### PR DESCRIPTION
**Pull request by dnfield**
Temporarily disable the test reporter logic, since it may be flaky.

@jonahwilliams

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: dev/bots/run_command.dart [hunk 1]
```diff
@@ -50,6 +50,7 @@ Stream<String> runAndGetStdout(String executable, List<String> arguments, {
     environment: environment,
   );
 
+  process.stderr.pipe(stderr);
```
Comment (jonahwilliams) [inline dev/bots/run_command.dart]: Is this what we were missing before?

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: dev/bots/run_command.dart (hunk 1 above)
Comment (dnfield) [reply dev/bots/run_command.dart]: That's the key part.

**Pull request merged**